db/migrate/20191206074550_add_exited_at_to_founders_table.rb

```diff
@@ -0,0 +1,36 @@
+class AddExitedAtToFoundersTable < ActiveRecord::Migration[6.0]
+  class User < ActiveRecord::Base
+    has_many :founders
+  end
+
+  class Founder < ActiveRecord::Base
+    belongs_to :startup
+    belongs_to :user
+
+    delegate :name, to: :user
+  end
+
+  class Startup < ActiveRecord::Base
+    has_many :founders
+  end
+
+  def change
+    add_column :startups, :exited_at, :datetime
+
+    Startup.reset_column_information
```

Comment:
Shouldn't `.reset_column_information` be called on all models used in the migration?